pkg/cmd/resources_cmds.go

```diff
@@ -155,13 +155,176 @@ func addAllResourcesCmds(rootCmd *cobra.Command) {
 	}, &Config)
 	resource.NewOperationCmd(rAccountsCmd.Cmd, "capabilities", "/v1/accounts/{account}/capabilities", http.MethodGet, map[string]string{}, &Config)
 	resource.NewOperationCmd(rAccountsCmd.Cmd, "create", "/v1/accounts", http.MethodPost, map[string]string{
-		"account_token":    "string",
-		"business_type":    "string",
-		"country":          "string",
-		"default_currency": "string",
-		"email":            "string",
-		"external_account": "string",
-		"type":             "string",
+		"account_token":                                            "string",
+		"business_profile.mcc":                                     "string",
+		"business_profile.name":                                    "string",
+		"business_profile.product_description":                     "string",
```

Comment:
IMO this is fine for a first pass, but in the medium/long term do you think we might want to either
(a) generate these arguments from protobufs
(b) have some sort of testing that helps us manually keep this list in sync

Reply:
For a - @etsai-stripe is doing so already in other places, so yes.
b - what are you imagining? I'm not sure I understand what this would do yet.

Reply:
If we have (a) to keep us in sync, we don't need (b) which would have simply alerted us to when we fell out of sync (manual action required to correct a hard-coded list of arguments)